server/test_integration/document.test.ts

```diff
@@ -185,7 +186,104 @@ describe("Integration Tests for Document API", () => {
     });
+
+
+    test("Test 9 - Should fetch multiple documents", async () => {
+        
+        const responsePost1 = await request(app)
+            .post('/api/documents')
+            .send({
+                title: "Document 1",
+                stakeholders: [Stakeholders.RESIDENT],
+                scale_info: Scale.TEXT,
+                scale: 10,
+                issuance_date: date,
+                type: KxDocumentType.INFORMATIVE,
+                language: "Swedish",
+                doc_coordinates: { type: AreaType.AREA, coordinates: [[KIRUNA_COORDS, KIRUNA_COORDS.map(c => c + 0.5), KIRUNA_COORDS.map(c => c - 0.5)]] },
+                description: "Test document 1"
+            });
+            documentIds.push(responsePost1.body._id);
+       const responsePost2 =  await request(app)
+            .post('/api/documents')
+            .send({
+                title: "Document 2",
+                stakeholders: [Stakeholders.RESIDENT],
+                scale_info: Scale.TEXT,
+                scale: 10,
+                issuance_date: date,
+                type: KxDocumentType.INFORMATIVE,
+                language: "Swedish",
+                doc_coordinates: { type: AreaType.AREA, coordinates: [[KIRUNA_COORDS, KIRUNA_COORDS.map(c => c + 0.5), KIRUNA_COORDS.map(c => c - 0.5)]] },
+                description: "Test document 2"
+            });
+    
+            documentIds.push(responsePost2.body._id);
+        const response = await request(app).get('/api/documents');
+    
+        
+        expect(response.status).toBe(200);
+        expect(response.body).toBeDefined();
+        expect(Array.isArray(response.body)).toBe(true);
+        expect(response.body.length).toBeGreaterThanOrEqual(2);
+        expect(response.body.some((doc: KxDocument) => doc.title === "Document 1")).toBe(true);
+        expect(response.body.some((doc: KxDocument) => doc.title === "Document 2")).toBe(true);
+    });
+
+    test("Test 10 - Insert 100 documents", async () => {
+        
+    
+        
+        for (let i = 0; i < 100; i++) {
+            const response = await request(app)
+                .post('/api/documents')
+                .send({
+                    title: `Document ${i + 1}`,
+                    stakeholders: [Stakeholders.RESIDENT],
+                    scale_info: Scale.TEXT,
+                    scale: 10,
+                    issuance_date: new Date(),
+                    type: KxDocumentType.INFORMATIVE,
+                    language: "Swedish",
+                    doc_coordinates: { type: AreaType.ENTIRE_MUNICIPALITY },
+                    description: `This is test document ${i + 1}.`,
+                });
+            expect(response.status).toBe(201);
+            documentIds.push(response.body._id);
+        }
+    
+        
+       
+    });
+
+    test("Test 10 - Insert 1000 documents", async () => {
+        
+    
+        
+        for (let i = 0; i < 1000; i++) {
```

Comment:
useless test

Reply:
for manual testing the ui

Reply:
it shouldn't be a test there is no reason for 1000 POST in the test, make it a curl one liner or anything similar